packages/components/src/Notification/Notification.component.js

```diff
@@ -2,11 +2,13 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import { TransitionGroup, CSSTransition } from 'react-transition-group';
 import classNames from 'classnames';
+import { translate } from 'react-i18next';
 
 import { Action } from '../Actions';
 import theme from './Notification.scss';
+import I18N_DOMAIN_COMPONENTS from '../constants';
 
-export function CloseButton({ notification, leaveFn }) {
+function CloseButtonComponent({ notification, leaveFn, t }) {
```

Comment:
eslint complaining about missing propTypes, I think we should add them.

Reply:
Yeah you're right, I have an eslint issue on my IDE :/
I add them